dashboard/app.py

```diff
@@ -7,24 +7,380 @@
 from dotenv import load_dotenv
 import dash
 
-app = Dash(__name__, use_pages=True)
-
-app.layout =html.Div(children=[
-    html.H1('DELOTON'),
-    html.Div(
-        [
-            html.Div(
-                dcc.Link(
-                     f"{page['name']} - {page['path']}", href=page["relative_path"]
-                )
-            )
-            for page in dash.page_registry.values()
-        ]
-    ),
-    dash.page_container
-])
+
+app = Dash(__name__)  # use_pages=True)
+
+app.layout = \
+    html.Div([
+        # Interval timers for updating graphs
+        dcc.Interval(
+             id='user-interval-component',
+             interval=60*60,
+             n_intervals=0
+             ),
+        dcc.Interval(
+            id='rides-interval-component',
+            interval=60*60*15,
```

Comment:
Aren't these intervals actually 60 minutes and 15 hours, not 60 seconds and 15 minutes?